<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>548</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Config dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>471</width>
     <height>491</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="envTab">
    <attribute name="title">
     <string>Environment</string>
    </attribute>
    <widget class="QGroupBox" name="audioGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>441</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>Audio</string>
     </property>
     <widget class="QWidget" name="layoutWidget1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>421</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="setupBgMusicLabel">
         <property name="text">
          <string>Setup background music</string>
         </property>
         <property name="buddy">
          <cstring>bgMusicPathLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="bgMusicPathLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="browseBgMusicButton">
           <property name="text">
            <string>Browse ...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="resetBgMusicButton">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>421</width>
        <height>18</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="enableEffectCheckBox">
         <property name="text">
          <string>Enable effects</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="enableLastWordCheckBox">
         <property name="text">
          <string>Enable last word</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="enableBgMusicCheckBox">
         <property name="text">
          <string>Enable background music</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>411</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="bgmVolumeLabel">
         <property name="text">
          <string>BGM    Volume</string>
         </property>
         <property name="buddy">
          <cstring>bgmVolumeSlider</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="bgmVolumeSlider">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget3_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>411</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="effectVolumeLabel">
         <property name="text">
          <string>Effect Volume</string>
         </property>
         <property name="buddy">
          <cstring>bgmVolumeSlider</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="effectVolumeSlider">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="bgGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>441</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Background</string>
     </property>
     <widget class="QWidget" name="layoutWidget4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>421</width>
        <height>73</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="backgrounSetupLabel">
           <property name="text">
            <string>Setup background</string>
           </property>
           <property name="buddy">
            <cstring>bgPathLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="bgPathLineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="browseBgButton">
           <property name="text">
            <string>Browse ...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="resetBgButton">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QCheckBox" name="noIndicatorCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>95</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>No indicator</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="noEquipAnimCheckBox">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>100</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>No equip anim</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="fullSkinCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Use full skin</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="noEffectsAnimCheckBox">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>120</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>No effects anim</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="fontSetupGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>441</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Font setup</string>
     </property>
     <widget class="QPushButton" name="changeAppFontButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Set application font</string>
      </property>
     </widget>
     <widget class="QLabel" name="appFontLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>24</y>
        <width>96</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Application font</string>
      </property>
      <property name="buddy">
       <cstring>appFontLineEdit</cstring>
      </property>
     </widget>
     <widget class="QLineEdit" name="appFontLineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>23</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="textEditFontLineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="setTextEditFontButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Font ...</string>
      </property>
     </widget>
     <widget class="QLabel" name="textEditFontLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>52</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Text edit font</string>
      </property>
      <property name="buddy">
       <cstring>textEditFontLineEdit</cstring>
      </property>
     </widget>
     <widget class="QPushButton" name="setTextEditColorButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Color ...</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="gameTab">
    <attribute name="title">
     <string>Game</string>
    </attribute>
    <widget class="QCheckBox" name="neverNullifyMyTrickCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Never nullify my single target trick</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="autoTargetCheckBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable auto target</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="intellectualSelectionCheckBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable intellectual selection</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="doubleClickCheckBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable double-click</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>106</y>
       <width>441</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="bubbleChatBoxLayout">
      <item>
       <widget class="QLabel" name="bubbleChatBoxKeepLabel">
        <property name="text">
         <string>Bubble Chat Box Keep Time</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="bubbleChatBoxKeepSpinBox">
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>2000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="superDragCheckBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable super-drag</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>441</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="replaysLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="replaysPathLabel">
          <property name="text">
           <string>Save path of replays</string>
          </property>
          <property name="buddy">
           <cstring>bgPathLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="saveNetworkOnlyCheckBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Save network only</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="replaysPathLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="autoSaveReplaysCheckBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>167</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable auto save replays</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>441</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="specifyGeneralLayout">
      <item>
       <widget class="QCheckBox" name="specifyGeneralCheckBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Enable specify general</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="generalNameLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="chooseGeneralButton">
        <property name="text">
         <string>Choose general ...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>510</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
